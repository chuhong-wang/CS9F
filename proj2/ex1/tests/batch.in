batch tests/commands.in
update word 100
