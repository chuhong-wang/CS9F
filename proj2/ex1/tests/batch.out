update word 5
update word 9
list names
list quantities
update word 100
